src/base/bittorrent/session.cpp

```diff
@@ -279,6 +279,7 @@ Session::Session(QObject *parent)
     , m_coalesceReadWriteEnabled(BITTORRENT_SESSION_KEY("CoalesceReadWrite"), false)
 #endif
     , m_isSuggestMode(BITTORRENT_SESSION_KEY("SuggestMode"), false)
+    , m_socketBacklogSize(BITTORRENT_SESSION_KEY("SocketBacklogSize"), 30)
```

Comment:
Why do you call it "Socket backlog size" unlike "Listen queue size" used in libtorrent?

Reply:
"listen" is a operation/function that operates on a socket and "backlog" is how the variable is really named.
http://man7.org/linux/man-pages/man2/listen.2.html